draft-ounsworth-csr-attestation.md

```diff
@@ -409,16 +411,21 @@ synchronized clocks.
 Epochs also require (unidirectional) communication.
 None of these things are practical when interacting with Hardware Security Modules (HSM).
 
-The definition of "fresh" is somewhat ambiguous in the context of CSRs, especially
+Additionally, the definition of "fresh" is somewhat ambiguous in the context of CSRs, especially
 considering that non-automated certificate enrollments are often asyncronous,
 and considering the common practice of re-using the same CSR
 for multiple certificate renewals across the lifetime of a key.
 "Freshness" typically implies both asserting that the data was generated
 at a certain point-in-time, as well as providing non-replayability.
+Also, HSMs are typically designed to not allow downgrade of private key storage
+properties; for example if a given key was asserted at time T to have been
+generated inside the hardware boundary and to be non-exportable,
+then it can be assumed that those properties of that key will continue
+to hold into the future, thus making freshness largely moot.
```

Comment:
```suggestion
to hold into the future.
```

Reply:
I see the use of this specification a bit broader and there freshness may be an important aspect, which is why we wrote the document about conveying a nonce in a certificate management protocol